moveit_planners/trajopt/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(moveit_planners_trajopt)
+
+# At least C++11 required for TrajOpt
+add_compile_options(-std=c++14)
+
+if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+find_package(Boost REQUIRED system filesystem date_time thread serialization)
```

Comment:
Remove, since we don't currently need Boost

Reply:
done